Source/WarpX.H

```diff
@@ -235,6 +235,7 @@ public:
     const amrex::MultiFab& getBfield_fp  (int lev, int direction) {return *Bfield_fp[lev][direction];}
     const amrex::MultiFab& getrho_fp (int lev) {return *rho_fp[lev];}
     const amrex::MultiFab& getF_fp (int lev) {return *F_fp[lev];}
+    const bool DoPML () const {return do_pml;};
```

Comment:
```suggestion
    bool DoPML () const {return do_pml;};
```
Oops, fix LGTM warning
> The 'const' modifier has no effect on return types